python/lsst/daf/butler/cli/cmd/__init__.py

```diff
@@ -19,9 +19,9 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
-__all__ = ["butler_import", "create", "config_dump", "config_validate", "query_collections",
-           "query_dataset_types"]
+__all__ = ["butler_import", "create", "config_dump", "config_validate", "prune_collection",
+           "query_collections", "query_dataset_types"]
```

Comment:
might make future changes easier if you make one per line so that a new one early in the alphabet doesn't cause multiple lines to reflow.